// @tenderly/actions/lib/actions.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Network = void 0;
/**
 * Networks supported by Web3 Actions
 */
var Network;
(function (Network) {
    Network["MAINNET"] = "mainnet";
    Network["GOERLI"] = "goerli";
    Network["SEPOLIA"] = "sepolia";
    Network["RSK"] = "rsk";
    Network["RSK_TESTNET"] = "rsk-testnet";
    Network["BSC"] = "bsc";
    Network["BSC_TESTNET"] = "bsc-testnet";
    Network["GNOSIS_CHAIN"] = "gnosis-chain";
    Network["POLYGON"] = "polygon";
    Network["MUMBAI"] = "polygon-mumbai";
    Network["OPTIMISTIC"] = "optimistic";
    Network["OPTIMISTIC_GOERLI"] = "optimistic-goerli";
    Network["ARBITRUM"] = "arbitrum";
    Network["ARBITRUM_GOERLI"] = "arbitrum-goerli";
    Network["ARBITRUM_NOVA"] = "arbitrum-nova";
    Network["AVA"] = "ava";
    Network["FUJI"] = "fuji";
    Network["FANTOM"] = "fantom";
    Network["FANTOM_TESTNET"] = "fantom-testnet";
    Network["CRONOS"] = "cronos";
    Network["CRONOS_TESTNET"] = "cronos-testnet";
    Network["BLAST"] = "blast";
    Network["BOBA_GOERLI"] = "boba-goerli";
    Network["BOBA_ETHEREUM"] = "boba-ethereum";
    Network["BOBA_BINANCE"] = "boba-binance";
    Network["BOBA_AVALANCHE"] = "boba-avalanche";
    Network["BOBA_AVALANCHE_FUJI"] = "boba-avalanche-fuji";
    Network["BOBA_BINANCE_RIALTO"] = "boba-binance-rialto";
    Network["BOBA_SEPOLIA"] = "boba-sepolia";
    Network["BASE"] = "base";
    Network["BASE_GOERLI"] = "base-goerli";
    Network["ZORA"] = "zora";
    Network["HOLESKY"] = "holesky";
    Network["ZORA_TESTNET"] = "zora-testnet";
    Network["ZORA_SEPOLIA"] = "zora-sepolia";
    Network["BASE_SEPOLIA"] = "base-sepolia";
    Network["ARBITRUM_SEPOLIA"] = "arbitrum-sepolia";
    Network["OPTIMISTIC_SEPOLIA"] = "optimistic-sepolia";
    Network["MANTLE"] = "mantle";
    Network["MANTLE_SEPOLIA"] = "mantle-sepolia";
    Network["LINEA"] = "linea";
    Network["LINEA_GOERLI"] = "linea-goerli";
    Network["LINEA_SEPOLIA"] = "linea-sepolia";
    Network["MOONBEAM"] = "moonbeam";
    Network["MOONRIVER"] = "moonriver";
    Network["FRAXTAL"] = "fraxtal-mainnet";
    Network["FRAXTAL_HOLESKY"] = "fraxtal-holesky";
})(Network = exports.Network || (exports.Network = {}));
